utils.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+
+# Build DAP test container & start the cluster
+function createDAPTestEnvironment() {
+  docker-compose build --pull client cuke-master test-dap
+  export CONJUR_APPLIANCE_URL="https://cuke-master"
+  docker-compose up -d client cuke-master test-dap
+
+  # Delay to allow time for conjur to come up
+  echo 'Waiting for conjur server to be healthy'
+  docker-compose run --rm test ./wait_for_server.sh
+}
+
+function loadDapTestPolicy() {
```

Comment:
Same as above, it looks like this method is only used in `test.sh`.

Reply:
Moved into test.sh